src/lib/datasource/FileReadDatasource.js

```diff
@@ -40,6 +40,21 @@ class FileReadDatasource extends Datasource {
         return Promise.resolve(null);
     }
 
+    mget(keysArray) {
+        return this.readFile().then((fileData)=> {
+            return keysArray.map((key) => fileData[key])
+                            .map(undefinedToNull);
+        });
+    }
+
+    mset(incomingObject) {
```

Comment:
thoese guys should be left undefined if they are not implemented.
I relate on that in Repository

Reply:
What you mean - should be left undefined?

Reply:
if method `mset` is not implemented, then `datasource.mset === undefined`
So just do not implement it if id does nothing.

Reply:
ok

Reply:
and, 'set' and 'delete' I will not implement, too?

Reply:
Yes. you can no implement them.